src/components/typography/Title.js

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
```

Comment:
Do katalogu `src/components/typography` dobrze jest dodać plik `index.js`, który będzie udostępniał na zewnątrz wszystko to co powinno być dostępne dla innych modułów (w tym przypadku Title powinien być dostępny).

Reeksport komponentów możesz zrobić tak:
```
export { default as Title } from './Title';
```